### ISSUE TITLE
Glitchy UI search results

### ISSUE THREAD

**Issue by Dendraspis** [0xd4d/dnSpy#1163]
_- What version did you use?_
v6.0.5 - Bug also appeared before and I'm kind of sure that it happened in v5, too

_- Write down all steps needed to reproduce this issue. Also attach any needed files._
From time to time this happens:
https://i.imgur.com/ArbkGr0.jpg

Reproducing is a little bit tricky. When I tried to reproduce it I used the search and analyse features very often and switched a few times. Mostly it appears directly after switching from Analyzer to Search after the first search afterwards.
Sorry, no better procedure possible, but I made a screenshot at least.

**Comment by 0xd4d** [comment]
I've never seen that before. Did you use the .NET Core build or the .NET Framework build?

**Comment by Dendraspis** [comment]
I believe you about that. It's the latest **.NET Framework** release. But as you ask I also tried the newest .NET CORE release and that's the result after about 50s of testing with multiple switches, multiple searches and multiple analyzes:
https://i.imgur.com/olcuI2L.jpg

In addition to that I had about 30-40 assemblies listed in the Assembly Explorer.

~~But I have noticed, that it sort of fixes itself. Really crazy.
For the example I searched for _speed_ most the time and some other words. After making the screenshot I wanted to know, if this bug holds on till the next restart. So I searched for _traffic_ - a word I used just a few times and the issue seems to appear a lot less. After a few very new word like _point_ and _vertex_ it was hard to find an issue. So I repeated my search for _speed_ and suddenly there is just 1 line (of about 800) has this problem.~~

Wow, a new approach I I'm kind of able to reproduce it - even without switching between Search and Analyzer:

1. Open dnSpy
1. Search for _speed_
1. Scroll the search results to the bottom (to simulate _nothing found_)
1. Search for something else, e.g. _traffic_
1. Scroll the search results to the bottom (to simulate _nothing found_)
1. Search for _speed_ again 

For me this leads to the broken UI issue - tested multiple times.
Interesting part is that the search result box scrolls to the top after something entered. But not when the bug occurs - then it scrolls to the middle:
https://i.imgur.com/i9dwovi.jpg

Hope this helps.
